modules/csharp/ROOT/examples/Program.cs

```diff
@@ -224,10 +225,10 @@ private static void TestClientCertAuthenticatorRootCerts()
             var auth = new ListenerCertificateAuthenticator(new X509Certificate2Collection(rootCert));
 
             // Create URL Endpoint Listener
-            var config = new URLEndpointListenerConfiguration(_Database);
-            config.DisableTLS = false; //The default value is false which means that the TLS will be enabled by default.
-            config.Authenticator = auth;
-            _listener = new URLEndpointListener(config);
+            var listenerConfig = new URLEndpointListenerConfiguration(_Database);
+            listenerConfig.DisableTLS = false; //The default value is false which means that the TLS will be enabled by default.
+            listenerConfig.Authenticator = auth;
+            _listener = new URLEndpointListener(listenerConfig);
```

Comment:
Update the listener config to `listenerConfig` because I used `config` for replicator config already.

Reply:
cool